README.md

```diff
@@ -24,9 +24,12 @@ As such, you must disable these controller loops in the `kube-controller-manager
 flag to `external` when starting the kube-controller-manager.
 
 For more details, please see:
-- https://github.com/kubernetes/community/blob/master/keps/sig-cloud-provider/0002-cloud-controller-manager.md
-- https://kubernetes.io/docs/tasks/administer-cluster/running-cloud-controller/#running-cloud-controller-manager
-- https://kubernetes.io/docs/tasks/administer-cluster/developing-cloud-controller-manager/
+
+- <https://github.com/kubernetes/community/blob/master/keps/sig-cloud-provider/0002-cloud-controller-manager.md>
+
+- <https://kubernetes.io/docs/tasks/administer-cluster/running-cloud-controller/#running-cloud-controller-manager>
+
+- <https://kubernetes.io/docs/tasks/administer-cluster/developing-cloud-controller-manager/>
```

Comment:
Is it required to have a blank line between each list item? Wouldn't it be better to add a blank line before the first item, but then keep the other lines together?

Reply:
yep... https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md032---lists-should-be-surrounded-by-blank-lines

Reply:
If I understand the rule correctly it should be possible to write:
```
For more details, please see:

- <https://github.com/kubernetes/community/blob/master/keps/sig-cloud-provider/0002-cloud-controller-manager.md>
- <https://kubernetes.io/docs/tasks/administer-cluster/running-cloud-controller/#running-cloud-controller-manager>
- <https://kubernetes.io/docs/tasks/administer-cluster/developing-cloud-controller-manager/>

## Examples
```